packages/expo-module-scripts/package.json

```diff
@@ -31,6 +31,9 @@
     "./jest-preset-plugin": {
       "default": "./jest-preset-plugin.js"
     },
+    "./jest-preset-plugin.js": {
+      "default": "./jest-preset-plugin.js"
+    },
```

Comment:
little unclear why to add `./jest-preset-plugin.js` here. do we have any code to import `jest-preset-plugin.js`?

Reply:
`et check-packages` complained with the message Mark had mentioned - and to remedy this I added the above. Not 100% sure if it is correct though, maybe there is some logic in jest that tries to import it?